.github/workflows/presubmit-workflow.yml

```diff
@@ -25,3 +25,50 @@ jobs:
         run: npm run build
         env:
           ELEVENTY_ENV: prod
+
+      # Checks to see if any files in the PR match one of the listed file types.
+      # We can use this filter to decide whether or not to run linters or tests.
+      # You can check if a file with a listed file type is in the PR by doing:
+      # if: ${{ steps.filter.outputs.md == 'true' }}
+      # This will return true if there's a markdown file that was changed
+      # in the PR.
+      - uses: dorny/paths-filter@v2.2.1
+        id: filter
+        with:
+          filters: |
```

Comment:
Since this only runs when we do presubmit, maybe we remove all the file path filtering stuff and just do lint and test?